gef-ui/server/utils/MOCKS/MOCK_FACILITIES.js

```diff
@@ -0,0 +1,178 @@
+const CONSTANTS = require('../../constants');
```

Comment:
Personal preference, totally up to you, but I think lowercase filenames is more friendly 😍 What do you think?

Reply:
changed